File: src/decoration/third/HouseBlend.java
package decoration.third;

import com.sun.jndi.ldap.Ber;

/**
 * Created by bvvy on 2017/7/19.
 */
public class HouseBlend extends Beverage {

    public HouseBlend() {
        description = "好吃的houseblend coffee";
    }

    @Override
    public float cost() {
        return 12.5f;
    }
}

File: src/decoration/second/HouseBlend.kt
package decoration.second

/**
 * Created by bvvy on 2017/7/16.
 */
class HouseBlend : Beverage() {
    init {
        description = "超级好吃的houseBlend咖啡"
    }

    override fun cost():Double {
        return 25.0
    }

}